web/css/tour.css

```diff
@@ -431,34 +438,79 @@ p.wv-tour-page-count {
   pointer-events: none;
 }
 
+.joyride-tip-guide.bordered span.joyride-nub::after {
+  content: " ";
+  display: block;
+  position: absolute;
+  width: 0;
+  height: 0;
+  z-index: -1;
+}
+
 .joyride-tip-guide span.joyride-nub.top {
   border-color: #afb5c4 !important;
   border-top-color: transparent !important;
   border-left-color: transparent !important;
   border-right-color: transparent !important;
 }
 
+.joyride-tip-guide.toolbarTip span.joyride-nub.top {
+  left: 250px;
+}
+
+.joyride-tip-guide.bordered span.joyride-nub.top::after {
+  top: -16px;
+  left: -15px;
+  border: 15px solid #808080 !important;
```

Comment:
Are all of these `!important` declarations necessary? These are pretty specific selectors already.

Reply:
Actually, that's a good point. On the after elements, `!important` should not be needed. I will double-check and remove.